core/src/kura.rs

```diff
@@ -305,224 +514,9 @@ pub enum Error {
     /// Inconsequential write
     #[error("Inconsequential block write.")]
```

Comment:
This error message is completely opaque. If we can, we should consider making it more obvious.